### DISCUSSION TITLE
'appearance-none' class not working in select element

### DISCUSSION THREAD

**Discussion by miraj09** [tailwindlabs/tailwindcss#5101] [Help]
why `appearance-none` class is not working in **select** element?

**Comment by simonswiss** [comment]
Hey!

What specifically is not working for you? Do you have some code to share? You can see it working on this Tailwind Play:

https://play.tailwindcss.com/wmaerDB7f6

The second select has the `appearance-none` class applied, and you can see that the chevron (arrow pointing down on the right) is removed:

![image](https://user-images.githubusercontent.com/485747/127265962-bbffc77c-a3ac-4134-b2b5-6c83017732f9.png)
